src/main/resources/application.properties

```diff
@@ -1 +1,6 @@
+spring.jpa.hibernate.ddl-auto=update
+spring.datasource.url=jdbc:oracle:thin:@localhost:1522:orcl
+spring.datasource.username=catalin
+spring.datasource.password=catalin
```

Comment:
This will be good in future to encrypt the password, but for now we will live it like this.